charts/ccsm-helm/test/identity/golden/deployment.golden.yaml

```diff
@@ -0,0 +1,65 @@
+---
+# Source: ccsm-helm/charts/identity/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ccsm-helm-test-identity
+  labels:
+    app: camunda-cloud-self-managed
+    app.kubernetes.io/name: identity
+    app.kubernetes.io/instance: ccsm-helm-test
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/part-of: camunda-cloud-self-managed
+    app.kubernetes.io/version: "SNAPSHOT"
+    app.kubernetes.io/component: identity
+  annotations:
+    {}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: camunda-cloud-self-managed
+      app.kubernetes.io/name: identity
+      app.kubernetes.io/instance: ccsm-helm-test
+      app.kubernetes.io/managed-by: Helm
+      app.kubernetes.io/part-of: camunda-cloud-self-managed
+      app.kubernetes.io/component: identity
+  template:
+    metadata:
+      labels:
+        app: camunda-cloud-self-managed
+        app.kubernetes.io/name: identity
+        app.kubernetes.io/instance: ccsm-helm-test
+        app.kubernetes.io/managed-by: Helm
+        app.kubernetes.io/part-of: camunda-cloud-self-managed
+        app.kubernetes.io/version: "SNAPSHOT"
+        app.kubernetes.io/component: identity
+    spec:
+      containers:
+      - name: identity
+        image: "camunda/identity:SNAPSHOT"
+        env:
+          - name: SERVER_PORT
+            value: "8080"
+          - name: KEYCLOAK_URL
+            value: "http://ccsm-helm-test-keycl:80/auth"
+          - name: IDENTITY_AUTH_PROVIDER_BACKEND_URL
+            value: "http://ccsm-helm-test-keycl:80/auth/realms/camunda-platform"
+          - name: KEYCLOAK_SETUP_USER
+            value: admin
+          - name: KEYCLOAK_SETUP_PASSWORD
+            valueFrom:
+                secretKeyRef:
+                  name: ccsm-helm-test-keycloak
+                  key: admin-password
+        resources:
+          limits:
+            cpu: 2000m
+            memory: 2Gi
+          requests:
+            cpu: 600m
+            memory: 400Mi
+        ports:
+        - containerPort: 8080
+          name: http
+          protocol: TCP
```

Comment:
```suggestion
          protocol: TCP

```

Reply:
This is a golden file no need for this :)